tests/expr_and_series/var_test.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import annotations
+
+import narwhals.stable.v1 as nw
+from tests.utils import Constructor
+from tests.utils import ConstructorEager
+from tests.utils import assert_equal_data
+
+data = {"a": [1, 3, 2], "b": [4, 4, 6], "z": [7.0, 8, 9]}
```

Comment:
We should consider adding a test with nulls, both here and in `std`. This can be a follow up PR